doc_requirements.txt

```diff
@@ -1,5 +1,5 @@
-pygments==2.6.1
 sphinx>=2.2.0,<3.0
+numpydoc==1.1.0
```

Comment:
Doesn't look like the submodule change was committed?

Reply:
thanks